### PR TITLE
Amended a couple of things on the site

### PR DESCRIPTION
**Pull request by domaspoliakas**
Hello,

Saw a couple of small things on the site and thought I'll PR an amendment.

One question - I believe I saw that `fs2-io` is only compatible with node, is that correct? I could add that as a comment in there as well

### CODE REVIEW

**Comment by armanbilge** [comment]
>  I believe I saw that fs2-io is only compatible with node, is that correct?

`fs2-io` was originally JVM only, now it's both JVM and Node.js. But right, no browser support, if that's what you are referring to?

**Comment by domaspoliakas** [comment]
Yeah, that's right. I'll add a comment that it's only for node.

File: site/getstarted/install.md [hunk 1]
```diff
@@ -17,10 +17,13 @@ libraryDependencies += "co.fs2" %% "fs2-io" % "<version>"
...
 ```
 
-The fs2-core library is also supported on Scala.js:
+The fs2-core as well as fs2-io libraries are also supported on Scala.js:
 
 ```
 libraryDependencies += "co.fs2" %%% "fs2-core" % "<version>"
+
+// not that fs2-io for scala.js only supports the node environment
+libraryDependencies += "co.fs2" %%% "fs2-io" % "<version>"
```
Comment (armanbilge) [inline site/getstarted/install.md:26]:
```suggestion
libraryDependencies += "co.fs2" %%% "fs2-core" % "<version>"
libraryDependencies += "co.fs2" %%% "fs2-io" % "<version>" // Node.js only
```

Comment (domaspoliakas) [reply site/getstarted/install.md]: can't believe I had a typo in a one-line comment commit 🤦‍♂️

Comment (armanbilge) [reply site/getstarted/install.md]: Happens to the best of us. Thanks for updating the docs! Hope to see more projects using this stuff :)

Comment (domaspoliakas) [reply site/getstarted/install.md]: Likewise! I think all the stuff you've been working on is very exciting, and I can't wait to see how feral evolves!

**Pull request merged**